<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathListConfigWidget</class>
 <widget class="QWidget" name="PathListConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="KListWidget" name="pathsList">
        <property name="toolTip">
         <string>List of directories to lookup header files</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="vl">
        <item>
         <widget class="KPushButton" name="addButton">
          <property name="toolTip">
           <string>Select directory to be added to the list</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="list-add"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="delButton">
          <property name="toolTip">
           <string>Remove selected directory from the list</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="list-remove"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="moveUpButton">
          <property name="toolTip">
           <string>Move selected directory up</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="arrow-up"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="moveDownButton">
          <property name="toolTip">
           <string>Move selected directory down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="arrow-down"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="clearButton">
          <property name="toolTip">
           <string>Clear directories list</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="edit-clear-list"/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="vs">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
